.travis.yml

```diff
@@ -1,5 +1,6 @@
-sudo: false
 language: generic
+services: docker
```

Comment:
Can you move `services: docker` and `sudo: required` to just the MIPS64 matrix cell?